src/js/components/DateTimeDrop.js

```diff
@@ -80,7 +80,7 @@ export default class DateTimeDrop extends Component {
   _buildDateRows (state) {
     const { timeOfDay, value } = state;
     const start = moment(value).startOf('month').startOf('week').add(timeOfDay);
-    const end = moment(value).endOf('month').endOf('week').add(timeOfDay);
+    const end = moment(start).add(41, 'days').add(timeOfDay);
```

Comment:
Can you add a short comment on why '41' was chosen? Otherwise, it seems a little odd to the casual observer.